src/librustc_lexer/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 // We want to be able to build this crate with a stable compiler, so feature
 // flags should optional.
```

Comment:
```suggestion
// flags should be optional.
```
While we are at it...